fastlane/Fastfile

```diff
@@ -312,6 +311,44 @@ platform :ios do
     end
   end
 
+  desc 'Upload iOS Mattermost.ipa file to s3'
+  lane :upload_ipa_to_s3 do
+    ipa_path = lane_context[SharedValues::IPA_OUTPUT_PATH]
+
+    unless ENV['AWS_BUCKET_NAME'].nil? || ENV['AWS_BUCKET_NAME'].empty? || ENV['AWS_REGION'].nil? || ENV['AWS_REGION'].empty? || ipa_path.nil?
+      s3_region = ENV['AWS_REGION']
+      s3_bucket = ENV['AWS_BUCKET_NAME']
+      s3_folder = ''
+
+      if ENV['BRANCH_TO_BUILD'] == 'pr'
+        s3_folder = "#{ENV['AWS_FOLDER_NAME']}/#{ENV['BRANCH_TO_BUILD']}"
+      else
+        version_number = get_version_number(xcodeproj: './ios/Mattermost.xcodeproj', target: 'Mattermost')
+        build_number = get_build_number(xcodeproj: './ios/Mattermost.xcodeproj')
+        s3_folder = "#{ENV['AWS_FOLDER_NAME']}/#{version_number}/#{build_number}"
+      end
+
+      s3 = Aws::S3::Resource.new(region: s3_region)
+      ipa_obj = s3.bucket(s3_bucket).object("#{s3_folder}/Mattermost.ipa")
+      ipa_obj.upload_file("#{ipa_path}")
+
+      if ENV['BRANCH_TO_BUILD'] == 'pr'
+        plist_template = File.read('plist.erb')
+        plist_body = ERB.new(plist_template).result(binding)
+
+        plist_obj = s3.bucket(s3_bucket).object("#{s3_folder}/#{filename}.plist")
+        plist_obj.put(body: plist_body)
+
+        ios_plist_install_url = "itms-services://?action=download-manifest&url=https://#{s3_bucket}/#{s3_folder}/#{filename}.plist"
+
+        post_pr_message({:install_url => ios_plist_install_url})
```

Comment:
I believe the post_pr_message applies to both Android and iOS and is only one message that is posted with both links.

Reply:
See https://github.com/mattermost/mattermost-mobile/pull/2796#discussion_r284174545